crates/pop-common/src/helpers.rs

```diff
@@ -31,12 +32,57 @@ pub fn replace_in_file(file_path: PathBuf, replacements: HashMap<&str, &str>) ->
 	Ok(())
 }
 
+/// Parses the package name from the `Cargo.toml` file located in the specified node path.
+///
+/// # Arguments
+/// * `node_path` - The path to the node directory containing the `Cargo.toml` file.
+pub fn parse_package_name(node_path: &Path) -> Result<String, Error> {
+	let manifest = node_path.join("Cargo.toml");
```

Comment:
Can be simplified by using crate::manifest::from_path() which returns a strongly type Manifest object which can be used to get the package name in a much simpler way.

Also suggest moving this function to the manifest module. I am personally not a fan of the 'helpers' pattern as it just becomes a dumping ground for unstructured functions, when they could at least be logically grouped by module.

I also question if we even need this function at all now, as getting the package name from the Manifest should be as simple as manifest.package.name (https://docs.rs/cargo_toml/latest/cargo_toml/struct.Package.html#structfield.name)

Reply:
The function is not needed anymore, can be `let node_name = manifest.package().name();` where it was called.